modules/@apostrophecms/login/index.js

```diff
@@ -346,6 +348,22 @@ module.exports = {
         // it should be accessed via POST because the result
         // may differ by individual user session and should not
         // be cached
+        async whoami (req) {
```

Comment:
Please run `npx eslint . --fix` to correct minor whitespace and code style issues without pain 😄